pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -81,6 +86,11 @@ func endExecutionSpan(processTrace func(p *api.Payload), requestID string, endTi
 		},
 	}
 
+	if strings.ToLower(os.Getenv(captureLambdaPayloadEnvVar)) == "true" {
```

Comment:
We should use [strconv.ParseBool](https://pkg.go.dev/strconv#ParseBool) to support all used format for `true`/`false`.

Reply:
config is now used to get this env variable